modules/product/src/main/java/com/opengamma/strata/product/SecuritizedProductTrade.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (C) 2016 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.product;
+
+import com.opengamma.strata.basics.currency.Currency;
+import com.opengamma.strata.basics.market.ReferenceData;
+
+/**
+ * A trade that is directly based on a securitized product.
+ * <p>
+ * This defines a trade in a securitized product.
+ * A securitized product contains the structure of a financial instrument that is traded as a {@link Security}.
+ * See {@link SecuritizedProduct} to understand the difference between a security and a securitized product.
+ * <p>
+ * When trading securities, the standard trade type is {@link SecurityIdTrade}.
+ * That trade type relies on securities being looked up in {@link ReferenceData}.
+ * One use for trade types that implement {@code SecuritizedProductTrade} is to price
+ * and hold trades without needing to populate reference data, because the securitized
+ * product representation completely models the trade.
+ * <p>
+ * Implementations of this interface must be immutable beans.
+ */
+public interface SecuritizedProductTrade
+    extends ProductTrade {
+
+  /**
+   * Gets the product of the security that was traded.
+   * 
+   * @return the product
+   */
+  @Override
+  public abstract SecuritizedProduct getProduct();
+
+  /**
+   * Gets the quantity that was traded.
+   * <p>
+   * This will be positive if buying and negative if selling.
+   * 
+   * @return the quantity
+   */
+  public abstract long getQuantity();
+
+  /**
+   * Gets the price that was traded.
```

Comment:
Is this the total price of the trade or the unit price (trade price / quantity)?